### ISSUE TITLE
Any way to control the location of scroll of ipydatagrid

### ISSUE THREAD

**Issue by jobinnthomas** [bloomberg/ipydatagrid#493]
I am not able to programmatically control the ipydatagrid scroll bars.
Even if i select a range of rows that are not visible programatically, the scroll bars dont scroll down. 

Is there any way to control the position of the scroll bars ? Any way to enable automatic scrolling to selected content?

**Comment by martinRenou** [comment]
Thank you for opening an issue! 

> Is there any way to control the position of the scroll bars ?

I don't think we should implement this feature, the scroll bar being a UI thing we should not have the kernel control it.

> Any way to enable automatic scrolling to selected content?

This, on the other hand, could indeed be a nice feature!

**Comment by jobinnthomas** [comment]
Yes..thats fine by me. It more of a 'is needed' rather than would be nice to have.
